common/txmgr/confirmer.go

```diff
@@ -206,6 +209,9 @@ func (ec *Confirmer[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) sta
 	if err != nil {
 		return fmt.Errorf("Confirmer: failed to load EnabledAddressesForChain: %w", err)
 	}
+	if err = ec.stuckTxDetector.LoadPurgeBlockNumMap(ctx, ec.enabledAddresses); err != nil {
```

Comment:
I considered this to avoid another method the confirmer needs to call but this method requires a context. I wouldn't have anything to wire this up to during initialization and didn't want to introduce a context to the TXM builder method just to pass it down to the detector.